lzy/allocator/src/main/java/ai/lzy/allocator/alloc/VmAllocator.java

```diff
@@ -40,6 +41,16 @@ public String message() {
         public Result withReason(String reason) {
             return new Result(code, reason);
         }
+
+        public static Result fromGrpcStatus(Status status) {
+            return switch (status.getCode()) {
+                case UNAVAILABLE, INTERNAL, RESOURCE_EXHAUSTED, UNKNOWN, ABORTED, DEADLINE_EXCEEDED ->
+                        VmAllocator.Result.RETRY_LATER.withReason(status.getDescription());
+                case INVALID_ARGUMENT, FAILED_PRECONDITION, PERMISSION_DENIED, UNAUTHENTICATED, UNIMPLEMENTED ->
+                        VmAllocator.Result.FAILED.withReason(status.getDescription());
+                default -> VmAllocator.Result.SUCCESS;
```

Comment:
which codes are allowed here?

Reply:
I've rewrited it without default case. So now it covers all cases explicitly. I left just two cases to be successful result - OK and ALREADY_EXISTS.